.github/workflows/dev-cluster-check.yml

```diff
@@ -63,6 +64,20 @@ jobs:
           to: openbridge-dev@redhat.com
           from: Dev cluster CI
           body: Dev cluster CI check failed. Please analyze the issue on https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/actions/workflows/dev-cluster-check.yml and implement a proper fix.
+      - name: Get Sheriff Name
+        if: ${{ failure() }}
+        run: |
+          chmod +x ./.github/scripts/sheriff-queue.sh
```

Comment:
I think it is possible to add +x to the file in git. It is not necessary to add +x with each run of the Github action.